pilot/pkg/model/context.go

```diff
@@ -363,7 +363,7 @@ const (
 	ServiceClusterName = "istio-proxy"
 
 	// DiscoveryPlainAddress discovery IP address:port with plain text
-	DiscoveryPlainAddress = "istio-pilot:15007"
+	DiscoveryPlainAddress = "istio-pilot:15010"
```

Comment:
??

Reply:
also the default we use in our helm/istio/templates/configmap.yaml